packages/config/config/devices/0x015f/mh-s511.json

```diff
@@ -0,0 +1,124 @@
+// McoHome Technology Co., Ltd. H-S511
+// Touch Panel Switch 1-Way
+{
+	"manufacturer": "McoHome Technology Co., Ltd.",
+	"manufacturerId": "0x015f",
+	"label": "MH-S511",
+	"description": "Touch Panel Switch 1-Way",
+	"devices": [
+		{
+			"productType": "0x5111",
+			"productId": "0x1406"
+		},
+		{
+			"productType": "0x5111",
+			"productId": "0x5103"
+		},
+		{
+			"productType": "0x5131",
```

Comment:
This might be a S513 instead? There is a pattern to these productTypes (5111 for S511, 5121 for S512, 5131 for S513)

Reply:
Agreed